_posts/2020-04-08-secure-software-development-lifecycle.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: How to implement secure software development lifecycle
+author: Rinorragi
+excerpt: Sharing my experience and thoughts about secure software development lifecycle
+tags:
+- infosec
+- software security
+- devsecops
+---
+
+## What is secure software development lifecycle
+
+Have you ever found youself wondering if the system you are implementing is secure enough? I have. Quite often actually. It is not an easy question to answer unless you are prepared. This blog post is about how to prepare yourself for that question. The short answer is Secure Software Development Lifecycle which I will call SSDLC from this point onwards. 
+
+You should be familiar on how developing software is divided into different phases. There can be more or there can be less depending on the chosen software project management framework in use. Phases can exists in short iterations or in a one long waterfall or anything between. These phases could be for example:
+
+![ssdlc phases](/img/ssdlc/phases.png)
+
+What SSDLC does is to put security requirements for each of the phases. This leads you to consider security in all phases of the software project. 
```

Comment:
"What SSDLC does is to put security requirements for each of the phases" tän lauseen voisi yksinkertaistaa esim. muotoon "SSDLC adds security requirements in each of the phases".

"This leads you to consider security in all phases of the software project. leads you to consider -> makes you consider